os/include/tinyara/reboot_reason.h

```diff
@@ -25,27 +25,27 @@
 
 typedef enum {
 	/* [Reboot Reason Code]
-	 * System : 0 ~ 29,
-	 * Network : 30 ~ 59,
-	 * Common Service : 60 ~ 89,
-	 * App : 90 ~ 254
+	 * System : 50 ~ 79,
+	 * Network : 80 ~ 109,
+	 * Common Service : 110 ~ 139,
+	 * App : 140 ~ 254
 	 */
-	REBOOT_REASON_INITIALIZED          = 0,
-	REBOOT_SYSTEM_DATAABORT            = 1,	 /* Data abort */
-	REBOOT_SYSTEM_PREFETCHABORT        = 2,	 /* Prefetch abort */
-	REBOOT_SYSTEM_MEMORYALLOCFAIL      = 3,	 /* Memory allocation failure */
-	REBOOT_SYSTEM_WATCHDOG             = 4,	 /* Watchdog timeout */
-	REBOOT_SYSTEM_HW_RESET             = 5,	 /* HW power reset */
-	REBOOT_SYSTEM_USER_INTENDED        = 6,	 /* Reboot from user intention */
-	REBOOT_SYSTEM_BINARY_UPDATE        = 7,	 /* Reboot for Binary Update */
-	REBOOT_SYSTEM_BINARY_RECOVERYFAIL  = 8,	 /* Binary Recovery Fail */
+	REBOOT_REASON_INITIALIZED          = 50,
+	REBOOT_SYSTEM_DATAABORT            = 51, /* Data abort */
+	REBOOT_SYSTEM_PREFETCHABORT        = 52, /* Prefetch abort */
+	REBOOT_SYSTEM_MEMORYALLOCFAIL      = 53, /* Memory allocation failure */
+	REBOOT_SYSTEM_WATCHDOG             = 54, /* Watchdog timeout */
+	REBOOT_SYSTEM_HW_RESET             = 55, /* HW power reset */
+	REBOOT_SYSTEM_USER_INTENDED        = 56, /* Reboot from user intention */
+	REBOOT_SYSTEM_BINARY_UPDATE        = 57, /* Reboot for Binary Update */
+	REBOOT_SYSTEM_BINARY_RECOVERYFAIL  = 58, /* Binary Recovery Fail */
 
-	REBOOT_SYSTEM_DSLP_RESET           = 27, /* System wake up from deep sleep */
-	REBOOT_SYSTEM_SYS_RESET_CORE       = 28, /* System reset by Core */
-	REBOOT_SYSTEM_BOD_RESET            = 29, /* Brownout reset */
+	REBOOT_SYSTEM_DSLP_RESET           = 77, /* System wake up from deep sleep */
+	REBOOT_SYSTEM_SYS_RESET_CORE       = 78, /* System reset by Core */
+	REBOOT_SYSTEM_BOD_RESET            = 79, /* Brownout reset */
```

Comment:
They are board specific but this header is the public, common header.
How about using this like below?
```h
in tinyara/reboot_reason.h
REBOOT_SYSTEM_BOARD_SPECIFIC1 = 77,
REBOOT_SYSTEM_BOARD_SPECIFIC2 = 78,
REBOOT_SYSTEM_BOARD_SPECIFIC3 = 79,

in amebad_reboot_reason.h
#include <tinyara/reboot_reason.h>

#define REBOOT_SYSTEM_DSLP_RESET  REBOOT_SYSTEM_BOARD_SPECIFIC1 
#define REBOOT_SYSTEM_SYS_RESET_CORE   REBOOT_SYSTEM_BOARD_SPECIFIC2
....
```